cmd/maya-agent/app/collector/collector.go

```diff
@@ -0,0 +1,178 @@
+package collector
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// openEBSMetrics is used to store the collected metrics
+type openEBSMetrics struct {
```

Comment:
Should it be VolumeMetrics ?

Reply:
Should it be present in types/v1/metrics.go, etc ?

Reply:
Can there be a case where this struct might be used by some other component ?

Reply:
Can a separate issue be raised that explains each property in detail. The unit associated with each Property.

Reply:
Done